FIX: Issue when only lumberjack pagetype is not defaultChild #7

christopherdarling opened the pull request:
Fixes issue #7

When we only have a single pagetype we don't use a dropdownfield so the onchange event in javascript/GridField.js is never triggered to update the state.pageType value.

micmania1 commented:
Hey, thanks for the PR. Could you instead add this into the if statement above which already does the count() check?

christopherdarling committed:
FIX: Issue when only lumberjack pagetype is not defaultChild #7

Fixes issue #7

When we only have a single pagetype we don't use a dropdownfield so the onchange event in javascript/GridField.js is never triggered to update the state.pageType value.

christopherdarling commented:
Hey, sure no problem - PR updated

micmania1 commented:
Thanks :+1:
